### PR TITLE
Input field fix

### PR DESCRIPTION
**Pull request by masterrom**
Android Input Field Display Fix

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,6 +14,9 @@
     "react-native": "0.55.4",
     "react-native-camera": "^1.1.4",
     "react-native-elements": "^0.19.1",
+    "react-native-keyboard-avoid": "^1.2.0",
+    "react-native-keyboard-aware-scroll-view": "^0.6.0",
+    "react-native-keyboard-aware-view": "0.0.14",
```
Comment (GregTheGreek) [inline package.json]: you need all three?

File: src/screens/setup/create/CreateWalletName.js [hunk 2]
```diff
@@ -14,6 +15,44 @@ const ethers = require('ethers');
...
+    
+    // keyboardWillHide = (event) => {
+    //     Animated.parallel([
+    //         Animated.timing(this.keyboardHeight, {
+    //         duration: event.duration,
+    //         toValue: 0,
+    //         })
+    //     ]).start();
+    // };
+
```
Comment (GregTheGreek) [inline src/screens/setup/create/CreateWalletName.js]: if you don;'t need it delete it

File: package.json (hunk 1 above)
Comment (masterrom) [reply package.json]: Changing it

**Comment by masterrom** [comment]
Requested Changes implemented

**Pull request merged**